src/hooks/useUnifiedCOGS.tsx

```
@@ -0,0 +1,94 @@
+        dailyCOGS = financialCosts.dailyCosts.map((d) => ({
+          date: d.date,
+          amount: d.total_cost,
+        }));
+        break;
+
+      case 'combined': {
+        totalCOGS = inventoryCosts.totalCost + financialCosts.totalCost;
+
+        // Merge daily data by date
+        const dateMap = new Map<string, number>();
+        inventoryCosts.dailyCosts.forEach((d) =>
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)